go/enclave/rpc_server.go

```diff
@@ -122,8 +122,12 @@ func (s *RPCServer) SubmitL1Block(_ context.Context, request *generated.SubmitBl
 }
 
 func (s *RPCServer) SubmitTx(_ context.Context, request *generated.SubmitTxRequest) (*generated.SubmitTxResponse, error) {
-	encryptedHash, err := s.enclave.SubmitTx(request.EncryptedTx)
-	return &generated.SubmitTxResponse{EncryptedHash: encryptedHash}, err
+	enclaveResponse := s.enclave.SubmitTx(request.EncryptedTx)
+	if enclaveResponse.Err != nil {
```

Comment:
I'm not sure about this. why not just send the response as it is to the rpc client?
That would have the benefit of cleaning up this rpc stuff so we can use a single wrapper for all methods.

Reply:
I'm not quite sure what do you mean by a single wrapper in this context?